sample-notificator-app/src/main/groovy/life/qbic/samplenotificator/DependencyManager.groovy

```diff
@@ -72,11 +70,8 @@ class DependencyManager {
         return createNotificationController
     }
 
-    EmailGenerator getEmailGenerator(){
-        String HTMLTemplatePath = "notification-template/email-update-template.html"
-        emailGenerator = new EmailGenerator(HTMLTemplatePath, notifications)
-        return emailGenerator
+    CreateNotificationPresenter getCreateNotificationsPresenter() {
```

Comment:
Maybe we can rename it? I think it probably is rather a controller than a presenter. What do you think?

```suggestion
    CreateNotificationPresenter getCreateNotificationsController() {
```

Reply:
As we discussed I'll rename it into getCreateNotificationConnector instead :)